geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/geotools/MetadataBackedDataStore.scala

```diff
@@ -218,47 +225,61 @@ abstract class MetadataBackedDataStore(config: NamespaceConfig) extends DataStor
     * @param schema new simple feature type
     */
   override def updateSchema(typeName: Name, schema: SimpleFeatureType): Unit = {
```

Comment:
Is there any sort of distributed locking that can be done around the metadata for these updates?

Reply:
yeah, I will add that

Reply:
oh, it's already there 6 lines down :)